src/examples/anchor/AnchorDanger.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Anchor } from '@zendeskgarden/react-buttons';
+
+const Example = () => {
+  return <Anchor isDanger>Go to the Garden</Anchor>;
```

Comment:
Is it helpful to provide an href declaration in the examples?

```
<Anchor isDanger href="#">
  Go to the Garden
</Anchor>
```

Reply:
Without the `href` the element won't be within the tab order. I would include it for all examples.